modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasksTask.path

```diff
@@ -43,12 +43,12 @@
 
 <tr>
 	<td>PREVIEW_TITLE</td>
-	<td>//span[@class='h3 task-content-title']</td>
+	<td>//*[contains(@class, 'task-content-title')]</td>
 	<td></td>
 </tr>
 <tr>
 	<td>PREVIEW_CONTENT</td>
-	<td>//span[@class='h3 task-content-title']/../div[2] | //span[@class='h3 task-content-title']/../../div</td>
+	<td>//*[contains(@class, 'web-content') or contains(@class, 'dynamic') or contains(@class, 'blogs') or contains(@class, 'kaleo-process') or contains(@class, 'comments')]/../../../following-sibling::div | //*[contains(@class, 'documents-and-media')]/../following-sibling::span[contains(@class, 'taglib-text')]</td>
```

Comment:
The preview content is different for each component's content, here I am making it work for all our tests